platforms/software/testing-base/src/main/java/org/gradle/api/tasks/testing/AbstractTestTask.java

```diff
@@ -641,6 +641,16 @@ public AbstractTestTask setTestNameIncludePatterns(List<String> testNamePattern)
         return this;
     }
 
+    /**
+     * Sets the JUnit Platform selector patterns to be used for test discovery.
+     * See the JUnit Platform documentation for more information.
+     */
+    @Option(option = "select-tests", description = "Sets the JUnit Platform selector patterns (requires JUnitPlatform to be used).")
+    public AbstractTestTask setJunitPlatformSelectorPatterns(List<String> junitPlatformSelectorPatterns) {
```

Comment:
:question: Shouldn't this rather be part of `JUnitPlatformOptions`?

Reply:
I was unsure whether `@Option` would work there.